### DISCUSSION TITLE
Updating FBM order status

### DISCUSSION THREAD

**Discussion by Kernel1453** [amzn/selling-partner-api-models#3037] [General]
Hi All,

We are getting all orders with Orders API. We see an order is pending. Then Approved. Now we want to Confirm or Cancel Order and then update shipment status and Carrier & TrackingID for fbm orders. Which api can help us? we are trying Feed API, Order UpdateShipmentStatus but we could not overcome this.
For any example we'll appreciated.

Thanks.

**Comment by ivancortesromero** [comment]
@Kernel1453 you were looking for [confirmShipment](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#post-ordersv0ordersorderidshipmentconfirmation)
